### DISCUSSION TITLE
meta-analysis using mikropml?

### DISCUSSION THREAD

**Discussion by marwa38** [SchlossLab/mikropml#309] [Q&A] [answered]
Hello guys 
Is there a feature in mikropml to conduct a meta-analysis?
Cheers
M

**Comment by kelly-sovacool** [comment, marked as answer]
Hi @marwa38,

Meta-analyses are not easy to automate. We would recommend starting from our example snakemake template (https://github.com/SchlossLab/mikropml-snakemake-workflow) and modifying it for your needs.
